guides/common/modules/proc_distribution-of-archived-content-view-versions.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Distribution_of_Archived_Content_View_Versions_{context}"]
+= Distribution of Archived Content View Versions
```

Comment:
Maybe "Distributing Archived Content View Versions"? in title, anchor, and file name.